index.js

```diff
@@ -16,9 +16,12 @@ module.exports = {
     "function-linear-gradient-no-nonstandard-direction": true,
     "keyframe-declaration-no-important": true,
     "media-feature-name-no-unknown": true,
+    "no-descending-specificity": true,
+    "no-duplicate-selectors": true,
     "no-empty-source": true,
     "no-extra-semicolons": true,
     "no-invalid-double-slash-comments": true,
+    "no-unknown-animations": true,
```

Comment:
As per the discussion, please remove this rule as it only considers animations defined _within the same stylesheet_ as known.